src/tests/GC/Stress/Framework/ReliabilityFramework.cs

```diff
@@ -161,6 +163,11 @@ public static int Main(string[] args)
 
         ReliabilityFramework rf = new ReliabilityFramework();
         rf._logger.WriteToInstrumentationLog(null, LoggingLevels.StartupShutdown, "Started");
+
+        Console.CancelKeyPress += (object sender, ConsoleCancelEventArgs args) => {
```

Comment:
NIT: Since you aren't using either of the parameters of the delegate, you could easily set them to `_` to indicate that they aren't used i.e.,

``csharp
Console.CancelKeyPress += (object _, ConsoleCancelEventArgs _) => { .. }
``